Sources/WeaverDI/DependencyContainer.swift

```diff
@@ -26,15 +26,6 @@ open class DependencyContainer {
         registerDependencies(in: self)
     }
     
-    public init(_ parent: DependencyContainer? = nil) {
```

Comment:
This init can't be deleted. That's actually what's making the CI fail. You'll have to write something like this:
```swift
public init(_ parent: DependencyContainer? = nil) {
    self.parent = parent
    instances = ThreadSafeInstanceCachingDecorator(cache: InstanceCache())
    builders = ThreadSafeBuilderStoringDecorator(builderStoring: BuilderStore())
    
    ...
}
```

Reply:
Ok, but I did it as a `convenience init` so that the class has only one `init` to be modified on new changes and avoid repetitive code. Ok?

Reply:
The CI still doesn't pass because the sample project doesn't compile. I spotted two issues:
- The two files you added are missing from the generated pod target. You can fix this by running `pod install` in the `Sample` directory.
- The convenience `init` you created breaks the generated code compilation. You can see and fix this error by opening the `Sample.xcworkspace` file and compile. I suggest leaving the `init` how it was, unless you find a better way to deal with this issue.